composer.json

```diff
@@ -23,7 +23,7 @@
         "phpstan/phpstan": "0.12.88",
         "phpunit/phpunit": "^9.1",
         "predis/predis": "^1.1",
-        "rector/rector": "0.11.16",
+        "rector/rector": "0.11.20",
```

Comment:
I'm not sure how this got in here, maybe you are working off an old branch? Please rebase and force push.

Reply:
@MGatner : its (#4832) already merged  to develop branch

Reply:
and now, I have deleted the previous commit

Reply:
Thanks!